libraries-3/src/main/java/com/baeldung/mail/attachment/EmailAttachmentReceiver.java

```diff
@@ -0,0 +1,122 @@
+package com.baeldung.mail.attachment;
+
+import static java.util.Arrays.asList;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Properties;
+
+import javax.mail.Folder;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Multipart;
+import javax.mail.NoSuchProviderException;
+import javax.mail.Part;
+import javax.mail.Session;
+import javax.mail.Store;
+import javax.mail.internet.MimeBodyPart;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EmailAttachmentReceiver {
+
+
+
+
```

Comment:
We have some unnecessary whitespace here - let's remove it.

Reply:
> We have some unnecessary whitespace here - let's remove it.

Done